components/screens/BlogScreen/BlogScreen.js

```diff
@@ -1,12 +1,68 @@
+import { useIsFocused } from '@react-navigation/native';
 import React from 'react';
-import { View, Text } from 'react-native';
+import { useState, useEffect } from 'react';
+import { View, Text, ScrollView, Linking, Button } from 'react-native';
+import { Image } from 'react-native-elements';
+import InAppBrowser from 'react-native-inappbrowser-reborn';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { Styles } from './BlogStyles';
+
+var blogData = [];
 
 const BlogScreen = () => {
+
+  const [data, setData] = useState([]);
+  const isFocused = useIsFocused();
+
+  const getData = async () => {
+    const resp = await fetch("https://cascade-api.herokuapp.com/blog");
+    const data = await resp.json();
+    setData(data);
+  };
+  useEffect(() => {
+    if (isFocused) {
+      getData();
+
+    }
+
+  }, [isFocused]);
+
   return (
-    <View>
-      <Text>Blog View</Text>
-    </View>
+    <SafeAreaView>
+      <ScrollView>
+
+        {data.map(({ title, postDate, body, url }) => (
+          <View key={url}>
+            <Text key={title} style={Styles.title}>
+              {title}
+            </Text>
+            <Text style={Styles.date}
+              key={postDate}>{postDate}</Text>
+            <Image key={'rc34'}
+              source={{ uri: 'https://cbc-drupal-assets.s3.amazonaws.com/styles/blog_image/s3/blog/leafy%20and%20dank%20%283%29.JPG?R2Pc9TRNWsiPfO0asl_VSPULtFDI4Dqs&itok=O8FLO9W1' }}
+              style={Styles.image}
+
+            />
```

Comment:
I would recommend keeping formatting consistent throughout the file, like moving the closing tag for image one line up, and generally having them either at the end of the line or on the new line.